.github/workflows/dev_tag_retrieval_pr.yaml

```diff
@@ -0,0 +1,88 @@
+name: Dev Tag Retreival PR Generator
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "!*-*"
+jobs:
+  gen_dev_tag_retrieval_pr:
+    name: Generate Dev Tag Retrieval PR
+    # Only run this job if we're in the main repo, not a fork.
+    if: github.repository == 'vmware-tanzu/tanzu-framework'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+
+      - name: Fetch current version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
```

Comment:
This workflow should not run for patch releases, only for minor releases. So if the current version is a patch bump, we should stop here.

Reply:
Fixed with b9a350acad587e0f378d75d70e602ad91bced4f1 (https://github.com/vmware-tanzu/tanzu-framework/pull/1190/commits/b9a350acad587e0f378d75d70e602ad91bced4f1)